src/main/java/com/nerodesk/Launch.java

```diff
@@ -73,6 +73,20 @@ public static void main(final String... args) throws Exception {
     }
 
     /**
+     * Return passed port or run on default one 8080.
+     * @param args Command line args
+     * @return Application port.
+     */
+    private static int port(final String... args) {
```

Comment:
Well, it's the first time in my life that somebody complain that method it too short. For me method is too big when it's more than 7 lines. Main methods have tendency to grow so I'm trying to extract that soon enough.